src/Request/UpdateRequest.php

```diff
@@ -29,7 +29,7 @@ public function getBody()
         return [
             IProto::SPACE_ID => $this->spaceId,
             IProto::INDEX_ID => $this->indexId,
-            IProto::KEY => [$this->key],
+            IProto::KEY => is_array($this->key) ? $this->key : [$this->key],
```

Comment:
@nekufa I checked other requests which deal with keys:
https://github.com/tarantool-php/client/blob/master/src/Request/SelectRequest.php#L16
https://github.com/tarantool-php/client/blob/master/src/Request/DeleteRequest.php#L13
To keep it consistent, UpdateRequest's constructor should expect `array $key`, or other requests should be updated to work with both scalar and array arguments. Not sure which option is preferable ;)